perf_bench_test.go

```diff
@@ -640,9 +645,12 @@ func calculateRPS(vmNum int) int {
 	return vmNum * baseRPS
 }
 
-func bindVMsToSocket() error {
+// bindVMs binds VMs to *bindSocket and/or binds a VM to *profileCPUID.
```

Comment:
it is unclear what and/or means here because the function has not arguments. Please explain each scenario. 
Also, please replace the global variables, like *ProfileCPU, *bindSocket, with function arguments. Using global variables confuses a lot.

Reply:
Done